emsdk_manifest.json

```diff
@@ -289,6 +290,15 @@
     "activated_cfg": "PYTHON='%installation_dir%/bin/python3'",
     "activated_env": "EMSDK_PYTHON=%installation_dir%/bin/python3;SSL_CERT_FILE=%installation_dir%/lib/python3.7/site-packages/certifi/cacert.pem"
   },
+  {
+    "id": "python",
+    "version": "3.7.4-2",
+    "bitness": 64,
+    "arch": "aarch64",
+    "macos_url": "python-3.7.4-2-macos.tar.gz",
```

Comment:
Won't this need that architecture string embedded in the filename to disambiguate from the one right above?

How about `python-3.7.4-2-macos-aarch64.tar.gz`? 

Also this version seems now out of sync with the updated version in the script below (3.9.2)

Reply:
What do you think about attaching the `.tar.gz` and I can upload it before we land this?